cmd/notary/main.go

```diff
@@ -11,20 +10,25 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
+	"github.com/docker/docker/pkg/homedir"
 	"github.com/docker/notary/pkg/passphrase"
 	"github.com/docker/notary/version"
 )
 
-const configFileName string = "config"
-const defaultTrustDir string = ".notary/"
-const defaultServerURL = "https://notary-server:4443"
-const idSize = 64
+const (
+	configFileName   string = "config"
+	defaultTrustDir  string = ".notary/"
+	defaultServerURL        = "https://notary.docker.io"
```

Comment:
This is the default notary server in Docker, but for dev I think we just want to be able to point to our local instances so this should be `https://notary-server:4443`